safe/impact_function/test/test_postprocessors.py

```diff
@@ -218,6 +227,57 @@ def test_size_post_processor(self):
         impact_fields = impact_layer.dataProvider().fieldNameMap().keys()
         self.assertIn(feature_value_field['field_name'], impact_fields)
 
+    def test_productivity_post_processors(self):
+        """Test for productivity, prod cost, and prod value"""
+        impact_layer = load_test_vector_layer(
+            'impact',
+            'indivisible_polygon_impact.geojson',
+            clone_to_memory=True)
+        impact_layer.keywords['exposure_keywords'] = {
+            'exposure': 'population'
+        }
+        self.assertIsNotNone(impact_layer)
+
+        # Test the size post processor.
+        result, message = run_single_post_processor(
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->